app/src/main/java/com/poupa/vinylmusicplayer/provider/MusicPlaybackQueueStore.java

```diff
@@ -42,7 +43,7 @@
     public static final String DATABASE_NAME = "music_playback_state.db";
     public static final String PLAYING_QUEUE_TABLE_NAME = "playing_queue";
     public static final String ORIGINAL_PLAYING_QUEUE_TABLE_NAME = "original_playing_queue";
-    private static final int VERSION = 5;
+    private static final int VERSION = 6;
```

Comment:
For you info (off-topic): With the PR https://github.com/AdrienPoupa/VinylMusicPlayer/pull/492 (to remove the "excessive" storage access requirement) I'm trying to move the playlist support internally to a private preference-backed storage.

The next step for me is to apply the same underlying mechanism to queue stores (hence this class would be dropped). I'll probably wait for you to land first, but I want to give you some hints on my upcoming proposal.